observability/logging/cluster-logging-support.adoc

```diff
@@ -0,0 +1,73 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cluster-logging-support"]
+include::_attributes/common-attributes.adoc[]
+= Support
+:context: cluster-logging-support
+
+toc::[]
+
+include::snippets/logging-supported-config-snip.adoc[]
+include::snippets/logging-compatibility-snip.adoc[]
+include::snippets/log6x-loki-statement-snip.adoc[]
+
+{logging-uc} {for} is an opinionated collector and normalizer of application, infrastructure, and audit logs. You can use it to forward logs to various supported systems.
+
+{logging-uc} is not:
```

Comment:
Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording), we should use full sentences to introduce lists. In this case, you could write it to simply say something like this: 
```suggestion
{logging-uc} is not any of the following things:
```
If you can find a better word than "thing" in this case, or find a better wording feel free to use that instead. My point is only to make this introduction more of a full sentence

Reply:
I see now that it's also written that way [here](https://github.com/openshift/openshift-docs/pull/90433/files#diff-77d2d6227e93fb3d86b17208f6df68b5deaacc212cb0bf686e7d945c5109d4a2R15). I still think this should be fixed at least in this instance, but if you want to fix it in both instances, you can either do that in this PR or defer to a later one and have more time to think about a good wording choice.

Reply:
Mind if I defer and update as part of https://issues.redhat.com/browse/OBSDOCS-1747

Reply:
Sure thing!